pkgs/by-name/vz/vzic/package.nix

```
@@ -0,0 +1,45 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  pkg-config,
+  glib,
+  tzdata,
+  libical,
+}:
+stdenv.mkDerivation rec {
+  pname = "vzic";
+  version = "tzdata2024a";
+
+  src = fetchFromGitHub {
+    owner = "libical";
+    repo = "vzic";
+    rev = "4a87e2bed237585cbcf209507d0b82c023113633";
+    hash = "sha256-zRy62SJa8XUBPnoW/fgANKZaatNWqmXka4LgRUlbLzw=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    tzdata
+    glib
+    libical
+  ];
+  postPatch = ''
+    sed -ie "s|^OLSON_DIR.*|OLSON_DIR = ${version}|" ./Makefile
+    sed -ie "s|^PRODUCT_ID.*|PRODUCT_ID = -//NixOS//NONSGML Citadel calendar//EN|" ./Makefile
+    sed -ie "s|^TZID_PREFIX.*|TZID_PREFIX = /NixOS/Olson_%D_1/|" ./Makefile
+  '';
```

The Makefile in `vzic` uses [conditional assignment](https://www.gnu.org/software/make/manual/html_node/Conditional-Assignment.html). This means that they can simply be overridden by environment, avoiding patching the Makefile.

```suggestion
  OLSON_DIR = version;
  PRODUCT_ID = "-//NixOS//NONSGML Citadel calendar//EN";
  TZID_PREFIX = "/NixOS/Olson_%D_1/";
```